modules/govuk_solr6/files/ckan28.schema.xml

```diff
@@ -0,0 +1,188 @@
+<?xml version="1.0" encoding="UTF-8" ?>
```

Comment:
Perhaps this file could be added in a separate commit so that it's easier to see the significance of the other file changes in this commit.

Reply:
Yeah ok I might do that...